images/windows/scripts/build/Install-WDK.ps1

```diff
@@ -16,9 +16,8 @@ if (Test-IsWin19) {
     $wdkExtensionPath = "C:\Program Files (x86)\Windows Kits\10\Vsix\VS2019\WDK.vsix"
 } elseif (Test-IsWin22) {
     # SDK is available through Visual Studio
-    $wdkUrl = "https://go.microsoft.com/fwlink/?linkid=2249371"
-    $wdkSignatureThumbprint = "7C94971221A799907BB45665663BBFD587BAC9F8"
-    $wdkExtensionPath = "C:\Program Files (x86)\Windows Kits\10\Vsix\VS2022\*\WDK.vsix"
```

Comment:
Do we really want to stop using `VSIX` extensions on windows 2022 images?

Reply:
Yes, the extension path isn't created after installation of newer WDK as previous version . wdk.vsix extension will installed along with visual studio components .